### DISCUSSION TITLE
Character bug

### DISCUSSION THREAD

**Discussion by Normandyrs** [runelite/runelite#17762] [Q&A] [answered]
Hi guys
Just been playing some gotr and notice my character has the appaerence of another player
Right now im doing slayer in Raiments robes, 99 mining cape and d pickaxe but my accounts inventory is setup with my slayer gear. So visually my character appears to look like someone else. 
Ive tested it out on the official runescape client but the problem seems to only be with runelite
![Transmorph](https://github.com/runelite/runelite/assets/167546033/166b3947-1140-4352-8dc7-e32194b17d43)

**Comment by raiyni** [comment, marked as answer]
Sounds like you a hub plugin installed that let's you change your gear

**Reply by Normandyrs** [reply]
Oh man ive been playing around with a couple of plugins i completely forgot that one

**Reply by Normandyrs** [reply]
Thanks a bunch
